app-hitl/hitl/src/hitl/utils/post_process.py

```diff
@@ -24,7 +24,7 @@ def process_tasks(task_grid_with_scores: gpd.GeoDataFrame, pred_geojson_uri: str
     merged_preds_gdf = merge_labels_with_task_grid(
         preds_gdf, tasks_to_update)
     # update task status to "LABELED" where predictions fall in
-    task_ids_to_update = list(set(merged_preds_gdf["taskId"]))
+    task_ids_to_update = list(set(tasks_to_update["taskId"]))
```

Comment:
If we mark all tasks as labeled, then this line and line 28 become not necessary. We can let line 31 directly do the job. What do you think?

Reply:
Good call, just updated